compiler/rustc_middle/src/ty/structural_impls.rs

```diff
@@ -221,76 +224,83 @@ impl<'tcx> fmt::Debug for Region<'tcx> {
 // copy...), just add them to one of these lists as appropriate.
 
 // For things for which the type library provides traversal implementations
-// for all Interners, we only need to provide a Lift implementation:
+// for all Interners, we only need to provide a Lift implementation.
 TrivialLiftImpls! {
-     (),
-     bool,
-     usize,
-     u64,
+    (),
```

Comment:
add tidy alphabetical checks to these? `// tidy-alphabetical-start`

Reply:
ok